### PR TITLE
updated contributing.md

### LINKED ISSUE #672: [Feature] Update the Contributing.md file
### Description

As the workflow for closing old pr's has been developed, there is a need to update details in the contributing.md file for contributors . Please refer to pr #655

### Screenshots

_No response_

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Tushar98644**
## Fixes #672

pls review @rupali-codes @CBID2

### CODE REVIEW

**Review by rupali-codes — APPROVED** [review]
looks good

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Thank you for taking the time to contribute to our project. Please take a moment
 >
 > **Pull Requests having no issue associated with them will not be accepted. Firstly get an issue assigned, whether it's already opened or raised by you, and then create a Pull Request.**
 
+> **An automated process has been implemented to ensure timely management of Pull Requests (PRs) on this platform. PRs that have been open for a duration exceeding 45 days will be automatically closed.**
```
Comment (CBID2) [inline CONTRIBUTING.md:9]: [Proofreading] 
```suggestion
> **An automated process has been implemented to ensure timely management of Pull Requests (PRs) on this platform. PRs that have been open for a duration exceeding 45 days will be automatically closed, so please plan accordingly.**
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
You’re off to a great start @Tushar98644! :) Just fix the proofreading error I mentioned and you’re good to go! 😊👍🏾

**Comment by Tushar98644** [comment]
> You’re off to a great start @Tushar98644! :) Just fix the proofreading error I mentioned and you’re good to go! 😊👍🏾

done

**Review by CBID2 — APPROVED** [review]
Well done @Tushar9864! :) LGTM! :)

**Pull request merged**